### PR TITLE
Docs: Add some examples

### PR DESCRIPTION
**Pull request by debugger22**

### CODE REVIEW

File: docs/source/development-workflow.rst [hunk 1]
```diff
@@ -0,0 +1,824 @@
...
+        4.1.5  Create GitHub account
+        4.1.6  Cloning BinPy
+      4.2  Create separated branch
+      4.3  Code modification
+      4.4  Be sure that all tests of BinPY passes
+      4.5  Commit the changes
+      4.6  Writing commit messages
+      4.7  Create a patch file or pull request for GitHub
+      4.8  Updating your pull request
+      4.9  Synchronization with master `sympy/sympy`.
```
Comment (rajat974) [inline docs/source/development-workflow.rst]: I think BinPy :P

Comment (debugger22) [inline docs/source/development-workflow.rst]: Oh yeah. @rvraghav93 ;)

Thanks I'll update it.

Comment (rajat974) [inline docs/source/development-workflow.rst]: @debugger22  i m rajat974 :P :P

Comment (ragv) [inline docs/source/development-workflow.rst]: No I think, he meant, Its me who made this mistake :P :P

On Sat, Apr 19, 2014 at 8:43 PM, Rajat Aggarwal notifications@github.comwrote:

> In docs/source/development-workflow.rst:
> 
> > -    4  Workflow process
> > -      4.1  Create your environment
> > -        4.1.1  Install git
> > -        4.1.2  Install other software
> > -        4.1.3  Basic git settings
> > -        4.1.4  Advanced tuning
> > -        4.1.5  Create GitHub account
> > -        4.1.6  Cloning BinPy
> > -      4.2  Create separated branch
> > -      4.3  Code modification
> > -      4.4  Be sure that all tests of BinPY passes
> > -      4.5  Commit the changes
> > -      4.6  Writing commit messages
> > -      4.7  Create a patch file or pull request for GitHub
> > -      4.8  Updating your pull request
> > -      4.9  Synchronization with master `sympy/sympy`.
> 
> @debugger22 https://github.com/debugger22 i m rajat974 :P :P
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/BinPy/BinPy/pull/206/files#r11794091
> .

## 

Raghav Rv

**Comment by debugger22** [comment]
Merging this PR. I'll add more docs later.

**Pull request merged**